TINY-8102: Replace badly named APIs (#7588)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- New `imagepreview` dialog component, allowing preview and zoom of any image URL #TINY-8333
- New `editor.annotator.removeAll` API to remove all annotations by name #TINY-8195
- New `Resource.unload` API to make it possible to unload resources #TINY-8431

> - New `dispatch()` function has been added to replace the old `fire()` function in various APIs #TINY-8102

Reviewer: ```suggestion
- New `dispatch()` function to replace the now deprecated `fire()` function in various APIs #TINY-8102
```

- The Editor query command APIs will now return `false` or an empty string on removed editors #TINY-7829
- The `plugins` option now returns a `string` array instead of a space separated string #TINY-8455
- Replaced 'Powered by Tiny' link text with logo #TINY-8371

> - The `fire()` function of `tinymce.Editor`, `tinymce.dom.EventUtils`, `tinymce.dom.DOMUtils`, `tinymce.util.Observable` and `tinymce.util.EventDispatcher` is now deprecated. Use `dispatch()` function instead #TINY-8102

Reviewer 2: Do we need a changelog entry for the `schema.getWhitespaceElements` change?

- The Editor query command APIs will now return `false` or an empty string on removed editors #TINY-7829
- The `plugins` option now returns a `string` array instead of a space separated string #TINY-8455
- Replaced 'Powered by Tiny' link text with logo #TINY-8371
- Replaced the `fire()` function with a `dispatch()` in various APIs #TINY-8102

> - Renamed the `tinymce.html.schema.getWhiteSpaceElement()` API to `tinymce.html.schema.getWhitespaceElements()` #TINY-8102

Reviewer 3: ```suggestion
- Renamed the `tinymce.html.schema.getWhiteSpaceElements()` API to `tinymce.html.schema.getWhitespaceElements()` #TINY-8102
```

That typo in the old name made me think this was going to break RTC :P

- The Editor query command APIs will now return `false` or an empty string on removed editors #TINY-7829
- The `plugins` option now returns a `string` array instead of a space separated string #TINY-8455
- Replaced 'Powered by Tiny' link text with logo #TINY-8371

> - Renamed the `tinymce.html.schema.getWhiteSpaceElements()` API to `tinymce.html.schema.getWhitespaceElements()` #TINY-8102

Reviewer: ```suggestion
- Renamed the `getWhiteSpaceElements()` function to `getWhitespaceElements()` in the `Schema` API #TINY-8102
```
